.githooks/pre-commit/prettify.js

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
```

Comment:
If this is a bash script, should the file name be changed away from .js?

But if it's a bash script, will it work on windows? Is that a concern at all, @ryanguill?

Reply:
I can change this to be a bash script. I'm not sure if this will actually work on Windows or not.